src/internal/types/themeData.ts

```diff
@@ -0,0 +1,15 @@
+export type ThemeSaveState = {
+  history: any; // json object
+  index: number;
```

Comment:
We won't need to save this soon because hash is superseding it. Probably just remove.